internal/converter/internal/otelcolconvert/testdata/jaegerremotesampling.river

```diff
@@ -0,0 +1,42 @@
+otelcol.extension.jaeger_remote_sampling "default" {
+	grpc {
+		endpoint = ":14250"
+	}
+
+	http {
+		endpoint = ":5778"
+	}
```

Comment:
Why don't these match with the defaults anymore? Do we need a new issue to sync them?

Reply:
Not sure, let me look (I copied these over from the jaeger receiver testcase).

Reply:
Yeah, sorry for the confusion. I didn't realize the reason why we aliased the types in the package was that we could set up component-specific defaults. Fixing this now.

Reply:
More specifically, we set up our defaults as 

https://github.com/grafana/agent/blob/2603bda1660254f57f22e42ac3b47ab2ef58eb6e/internal/component/otelcol/extension/jaeger_remote_sampling/jaeger_remote_sampling.go#L41-L50

whereas our fork of the jaegerremotesampling processor sets them up as
https://github.com/grafana/agent/blob/2603bda1660254f57f22e42ac3b47ab2ef58eb6e/internal/component/otelcol/extension/jaeger_remote_sampling/internal/jaegerremotesampling/factory.go#L44-L52

Upstream uses this function to decide whether to add localhost or 0.0.0.0 up front, so in a sense our defaults _did_ drift, and I'll add a commit to fix it to 0.0.0.0.

https://github.com/grafana/agent/blob/2603bda1660254f57f22e42ac3b47ab2ef58eb6e/internal/component/otelcol/extension/jaeger_remote_sampling/jaeger_remote_sampling.go#L41-L50